<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="output">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>321</width>
      <height>74</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>168</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>teller A</string>
     </property>
     <property name="buddy">
      <cstring>tellerA</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="tellerA"/>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>184</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>noemer A</string>
     </property>
     <property name="buddy">
      <cstring>noemerA</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="noemerA"/>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>168</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>teller B</string>
     </property>
     <property name="buddy">
      <cstring>tellerA</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="tellerB"/>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>183</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>noemer B</string>
     </property>
     <property name="buddy">
      <cstring>noemerA</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="noemerB"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>462</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBestand">
    <property name="title">
     <string>Bestand</string>
    </property>
    <addaction name="actionActies"/>
    <addaction name="separator"/>
    <addaction name="actionAfsluiten"/>
   </widget>
   <addaction name="menuBestand"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAfsluiten">
   <property name="text">
    <string>Afsluiten</string>
   </property>
  </action>
  <action name="actionActies">
   <property name="text">
    <string>Acties</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
